### PR TITLE
[IMP] payment_provider/mercado pago: content updates

### PR DESCRIPTION
**Pull request by masi-odoo**
This commit will add important clarifications to use and test the Mercado Pago payment provider and add visual documentation similar to other provider's documentation

### CODE REVIEW

**Comment by masi-odoo** [comment]
Hello @samueljlieber thanks for that! Linking Google Drive folder with images: 

https://drive.google.com/drive/folders/1CuOVMkXwREj_H5yuKdJ1oPksQWoDJ6Nk?usp=sharing

**Comment by samueljlieber** [comment]
Hi @masi-odoo I've implemented your images and updated the formatting for your changes in https://github.com/odoo/documentation/commit/1d92f29c1e20564cff28216af3553a0bffbb2866.

File: content/applications/finance/payment_providers/mercado_pago.rst [hunk 1]
```diff
@@ -32,5 +41,14 @@ Configuration on Odoo
...
 #. Configure the rest of the options to your liking.
 
+.. image:: mercado_pago/mp-access-token.png
+   :align: center
+   :alt: Access token configuration for Mercado Pago in Odoo.
+
+.. important::
+   Mercado Pago functions correctly with the :guilabel:`Enabled` status activated for production and
+   testing purposes. For this payment provider the *Test Mode* configuration in Odoo will result in
+   an error message.
```
Comment (AntoineVDV) [inline content/applications/finance/payment_providers/mercado_pago.rst:51]: I was not aware of this but it's not normal. Please create a support ticket so we can fix it instead :)

**Review by AntoineVDV — CHANGES_REQUESTED** [review]
Hello, I left some review comments below :)

**Comment by samueljlieber** [comment]
Thank you for your review @AntoineVDV! I implemented your changes in https://github.com/odoo/documentation/commit/c00e36cbe06dbd290c146ececbc4d4bd5e3598bd and https://github.com/odoo/documentation/commit/b8dff5505f0ba63d08f3e2df567225a8881f96d0. I also added a seealso link to the latest [Mercado Pago Odoo webinar](https://www.youtube.com/watch?v=CX8vPHMb1ic) for @masi-odoo.

File: content/applications/finance/payment_providers/mercado_pago.rst [hunk 2]
```diff
@@ -21,6 +24,10 @@ Configuration on Mercado Pago Dashboard
    If you are trying Mercado Pago as a test, select :guilabel:`Credienciales de prueba` in the left
    part of the application page, then copy the test :guilabel:`Access token`.
 
+.. image:: mercado_pago/mp-credentials.png
+   :align: center
```
Comment (auva-odoo) [inline content/applications/finance/payment_providers/mercado_pago.rst:28]: We are moving away from center-aligned images so you can remove this.

File: content/applications/finance/payment_providers/mercado_pago.rst [hunk 3]
```diff
@@ -5,6 +5,9 @@ Mercado Pago
 `Mercado Pago <https://www.mercadopago.com/>`_ is an online payment provider covering several
 countries, currencies and payment methods in Latin America.
 
+.. seealso::
+   `Mercado Pago Odoo webinar <https://www.youtube.com/watch?v=CX8vPHMb1ic>`_
```
Comment (auva-odoo) [inline content/applications/finance/payment_providers/mercado_pago.rst:9]: I would move this at the end of the page and merge it with the other seealso (and have 2 bullet points in there).

**Review by auva-odoo — CHANGES_REQUESTED** [review]
Hey @masi-odoo thank you for you work! I just added 2 suggestions, let me know if you have any questions!

**Review by AntoineVDV — APPROVED** [review]

**Review by auva-odoo — APPROVED** [review]
Thanks @samueljlieber ! :)

@robodoo r+

**Pull request closed without merging**